.github/ISSUE_TEMPLATE/03-documentation.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Documentation
+about: Report any issue observed within the Chapel documentation or the `chpldoc` tool.
+title: '[Documentation]: '
+labels: ''
+---
+
+### Summary of Problem
+
+**Description:**
+<!--
+What did you observe when encountering this issue?
+What did you expect to observe?
+-->
+
+### Steps to Reproduce
+
+**Source Code:**
+
+```chapel
+// Please provide source code that will reproduce the problem.
+// You can insert your code inline if it's not too long.
+// Otherwise, you can attach it as a file or provide a URL to it.
+// To the extent possible, providing simplified programs demonstrating the
+// problem will be appreciated.
+// You can also replace this section with "Associated Future Test(s)" below.
+```
+
+**Compile command:**
+<!-- e.g. `chpldoc foo.chpl` -->
+
+**Additional steps:**
+<!-- Are there any additional steps needed to demonstrate the issue beyond calling `chpldoc`? -->
+
+**Associated Future Test(s):**
+<!--
+Are there any tests in Chapel's test system that demonstrate this issue?
+e.g. [`test/path/to/foo.chpl`](
+      https://github.com/chapel-lang/chapel/blob/main/test/path/to/foo.chpl
+      ) #1234
+-->
+
+### Configuration Information
+
+- Output of `chpldoc --version`:
+- Output of `$CHPL_HOME/util/printchplenv --anonymize`:
+- Back-end compiler and version, e.g. `gcc --version` or `clang --version`:
+- Sphinx version:
+- sphinxcontrib-chapeldomain version:
```

Comment:
> The best way I know about is to look in third-party/chpl-venv/chpldoc-requirements3.txt

Similarly, we can get Sphinx version in `$CHPL_HOME/third-party/chpl-venv/chpldoc-requirements2.txt`, right?

> Do you think we should do more to help people provide that information? Or just accept that people won't fill that part in themselves?

As you mentioned, it could be very nice to have all we need visible when calling `chpldoc --version`, even if I don't know the implementation complexity behind this feature. For the need of this PR, I propose to provide the following hints:

```
- Sphinx version, e.g. `grep -E "^Sphinx" $CHPL_HOME/third-party/chpl-venv/chpldoc-requirements2.txt`:
- sphinxcontrib-chapeldomain version, e.g. `grep -E "^sphinxcontrib-chapeldomain" $CHPL_HOME/third-party/chpl-venv/chpldoc-requirements3.txt`:
```

Or may be better:
```
- Sphinx version: <!-- e.g. `grep -E "^Sphinx" $CHPL_HOME/third-party/chpl-venv/chpldoc-requirements2.txt` -->
- sphinxcontrib-chapeldomain version: <!-- e.g. `grep -E "^sphinxcontrib-chapeldomain" $CHPL_HOME/third-party/chpl-venv/chpldoc-requirements3.txt` -->
```

What do you think?

Reply:
> Similarly, we can get Sphinx version in $CHPL_HOME/third-party/chpl-venv/chpldoc-requirements2.txt, right?

Yup, that's right.

Of those two options you propose, I'd go with the first one.  I don't think it'll make too much difference to hide the command from the final display